EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -68,18 +84,18 @@ public struct MediaPlayerView: View {
         }
     }
     
-    @ViewBuilder private func renderForward() -> some View {
+    @ViewBuilder private func renderForward(disabled: Bool = false) -> some View {
         renderTimeElapsedWrapper {
-            renderButton(control: .forward)
+            renderButton(control: .forward, disabled: disabled)
         }
     }
     
-    @ViewBuilder private func renderButton(control: Control) -> some View {
+    @ViewBuilder private func renderButton(control: Control, disabled: Bool = false, color: Color = .white) -> some View {
         control.systemImage
-            .resizable()
-            .frame(width: Constants.iconSize.width, height: Constants.iconSize.height)
-            .foregroundColor(.white)
+            .font(.system(size: Constants.iconSize))
+            .tint(color)
             .wrapInButton(action: { viewModel.onTapAction?(control) })
+            .disabled(disabled)
     }
```

Comment:
For now yes, just added it as a parameter in case it would be used for something ahead, like the stuff @pedromotita  talked about disabling the rewind at the start...

I changed the frame for a font because the icons weren't adjusting well with the frame itself, and SF Symbols have their sizes adjusted by font, so making it this way we guarantee the icons don't get distorted

Reply:
Oh I see. I'd still leave the frame there (after the font) to ensure that the icons don't expand more than expected (and definitely rename the font constant to `iconFontSize`)

Reply:
And regarding the `disabled` modifier, you can still add it specifically to where you'd use it:
```swift
renderStuff()
    .disabled(true)

// for

@ViewBuilder private func renderStuff() -> some View {
    renderMoreStuff()
}

// instead of

renderStuff(disabled: true)

// for

@ViewBuilder private func renderStuff(disabled: Bool) -> some View {
    renderMoreStuff()
        .disabled(disabled)
    }
}
```